consensus/istanbul/backend/engine.go

```diff
@@ -629,12 +629,10 @@ func (sb *Backend) updateReplicaStateLoop(bc *ethCore.BlockChain) {
 	for {
 		select {
 		case chainEvent := <-chainEventCh:
-			sb.coreMu.RLock()
 			if !sb.isCoreStarted() && sb.replicaState != nil {
 				consensusBlock := new(big.Int).Add(chainEvent.Block.Number(), common.Big1)
 				sb.replicaState.NewChainHead(consensusBlock)
 			}
-			sb.coreMu.RUnlock()
 		case err := <-chainEventSub.Err():
```

Comment:
This removal of the lock looks safe to me since the locking was added in 1188111ac106a92c6c9d89c5ff3da265b404a161 to prevent a data race on `coreStarted` and `coreStarted` has since become an `atomic.Value`